.travis.yml

```diff
@@ -35,6 +33,8 @@ env:
     - SCRIPTED_TEST="scripted project-load/*"
     - SCRIPTED_TEST="safeUnitTests"
     - SCRIPTED_TEST="checkBuildScala211"
+    # - SCRIPTED_TEST="mavenResolverPluginTest:scripted dependency-management/*1of2 project/transitive-plugins"
```

Comment:
as @dwijnand pointed to my commit message, 50% of Travis CI failures are coming from `mavenResolverPluginTest:scripted`, and it seems like many of them are flaky (goes away after repeatedly restarting the build jobs). We certainly should figure out why that is and fix them.
#1921 likely need refixing, since I'm seeing local failures. Also adding Maven cache files into Travis cache may help.

At the moment, however, it's very labor intensive to validate the PRs because of the high false positive rate. This would result to either us ignoring Travis failures and missing legitimate bugs, or not merging the PRs in timely manner. Neither of which are good tradeoff with keeping flaky tests in our build.